testutil/singlevm/cleanup.sh

```diff
@@ -3,12 +3,17 @@
 . ~/local/demo.sh
 
 ciao_gobin="$GOPATH"/bin
+ciao_host=$(hostname)
 sudo killall ciao-scheduler
 sudo killall ciao-controller
 sudo killall ciao-launcher
 sleep 2
 sudo "$ciao_gobin"/ciao-launcher --alsologtostderr -v 3 --hard-reset
-sudo iptables -D FORWARD -p all -i ciao_br -j ACCEPT
+sudo iptables -D FORWARD -i ciao_br -j ACCEPT
+sudo iptables -D FORWARD -i ciaovlan -j ACCEPT
+if [ "$ciao_host" == "singlevm" ]; then
+	sudo iptables -D FORWARD -i ens2 -j ACCEPT
```

Comment:
I'm not seeing it in the code elsewhere, but is there something for a systemd/udev rule that insures the interface is named "ens2"?